site/content/en/docs/Examples/custom-controller.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Custom Controller for Agones Game Servers"
+linkTitle: "Custom Controller"
+date:
+publishDate:
+description: >
+  This Custom Controller example shows how to create, deploy and run a Custom Kubernetes Controller for Agones that logs changes to GameServers and modifies their labels.
+---
+
+## Prerequisite
+
+ To get started, ensure the following prerequisites are met:
+
+  - You have a running Kubernetes cluster.
+  
+  - Agones is installed on your cluster. See [Agones guide](https://agones.dev/site/docs/installation/install-agones/).
+
+  - Example custom controller code is available {{< ghlink href="examples/custom-controller" >}}here{{< /ghlink >}}
```

Comment:
Actually - also realised, assuming https://github.com/googleforgames/agones/pull/3725/files#r1548758608 works --- there is no prerequ to have the code locally!

But it is nice to look at it. Maybe:

```suggestion
 - (Optional) Review {{< ghlink href="examples/custom-controller" >}}example code{{< /ghlink >}} to see the details of this example.
```

... or something like that? Not sure I love the wording.